packages/block-editor/src/components/block-allowed-blocks/allowed-blocks-control.js

```diff
@@ -0,0 +1,85 @@
+/**
+ * WordPress dependencies
+ */
+import { BaseControl, Button } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { useState } from '@wordpress/element';
+import { useSelect } from '@wordpress/data';
+import { hasBlockSupport, store as blocksStore } from '@wordpress/blocks';
+import { useInstanceId } from '@wordpress/compose';
+
+/**
+ * Internal dependencies
+ */
+import BlockAllowedBlocksModal from './modal';
+import { store as blockEditorStore } from '../../store';
+
+export default function BlockAllowedBlocksControl( { clientId } ) {
+	const [ isBlockControlOpened, setIsBlockControlOpened ] = useState( false );
+	const instanceId = useInstanceId(
+		BlockAllowedBlocksControl,
+		'allowed-blocks-control'
+	);
+
+	const { blockTypes, selectedBlockNames } = useSelect(
+		( select ) => {
+			const { getBlockAttributes } = select( blockEditorStore );
+			return {
+				blockTypes: select( blocksStore ).getBlockTypes(),
+				selectedBlockNames:
+					getBlockAttributes( clientId )?.allowedBlocks,
+			};
+		},
+		[ clientId ]
+	);
+
+	const filteredBlockTypes = blockTypes.filter(
+		( blockType ) =>
+			hasBlockSupport( blockType, 'inserter', true ) &&
+			( ! blockType.parent ||
+				blockType.parent.includes( 'core/post-content' ) )
+	);
+
+	if ( ! filteredBlockTypes ) {
+		return null;
+	}
+
+	const selectedBlockTypes =
+		selectedBlockNames === undefined
+			? filteredBlockTypes
+			: filteredBlockTypes.filter( ( blockType ) =>
+					selectedBlockNames.includes( blockType.name )
+			  );
+
+	return (
+		<div className="block-editor-block-allowed-blocks-control">
+			<BaseControl
+				id={ instanceId }
+				label={ __( 'Allowed Blocks' ) }
```

Comment:
This is outputting a label that doesn't correspond to an actual element in the DOM:

<img width="321" height="360" alt="Screenshot 2025-10-08 at 10 37 28 am" src="https://github.com/user-attachments/assets/554353fa-193c-4b99-8bd2-7b505a32b9c8" />

Because the Button is already accessibly labeled, it's better to use [BaseControl.VisualLabel](https://github.com/WordPress/gutenberg/blob/trunk/packages/components/src/base-control/README.md#basecontrolvisuallabel) here instead of the label prop.

Reply:
Fixed in c4447a3e75876e0c8780eea92ab7e521ddbcd706